tests/Doctrine/Tests/DBAL/Platforms/AbstractMySQLPlatformTestCase.php

```diff
@@ -248,9 +248,9 @@ public function testChangeIndexWithForeignKeys() : void
     /**
      * @return string[]
      */
-    protected function getQuotedColumnInPrimaryKeySQL() : array
+    protected function getQuotedColumnInPrimaryKeySQL(string $columnName = 'create') : array
```

Comment:
Why does the parameter need to be optional?

Reply:
@greg0ire suggested it.  @greg0ire, I don't think this is needed - can I remove all occurrences?

Reply:
I suggested to add an optional argument iff you need to change that `create` value? Which you did. If you can make it non-optional and update all the pre-existing method call to pass `create`, that's great. If you won't need the value to change, that's great too.

Reply:
There was no need for it to be an optional argument because I had already updated all callers (including passing the "default" sometimes).  Fixed.

Reply:
@greg0ire, should I squash the commits again immediately?

Reply:
Yes

Reply:
Done.